main.js

```diff
@@ -48,3 +50,140 @@ navbarBtn.addEventListener('click', () => {
+  const img = document.createElement('img');
+  img.src = project.image;
+  cardImageDiv.appendChild(img);
+  cardDiv.appendChild(cardImageDiv);
+  const cardBodyDiv = document.createElement('div');
+  cardBodyDiv.className = 'card-body';
+  const BodyTitleDiv = document.createElement('div');
+  BodyTitleDiv.className = 'card-title';
+  const h2 = document.createElement('h2');
+  const h2Text = document.createTextNode(project.name);
+  h2.appendChild(h2Text);
+  BodyTitleDiv.appendChild(h2);
+  cardBodyDiv.appendChild(BodyTitleDiv);
+  const ClientInfoDiv = document.createElement('div');
+  ClientInfoDiv.className = 'card-client-info';
+  const ul = document.createElement('ul');
+  const itemlist = ['CANOPY', 'Back End Dev', '2015'];
+  const lists = repeatList(itemlist, 'info');
+  ul.appendChild(lists);
+  ClientInfoDiv.appendChild(ul);
+  cardBodyDiv.appendChild(ClientInfoDiv);
+
+  const BodyPragrphDiv = document.createElement('div');
+  BodyPragrphDiv.className = 'card-pragrph';
+  const p = document.createElement('p');
+  const pText = document.createTextNode(project.description);
+  p.appendChild(pText);
+  BodyPragrphDiv.appendChild(p);
+  cardBodyDiv.appendChild(BodyPragrphDiv);
+
+  const CardTechDiv = document.createElement('div');
+  CardTechDiv.className = 'card-tech';
+  const ulTech = document.createElement('ul');
+  const listTech = project.technologies;
+  const listTechs = repeatList(listTech);
+  ulTech.appendChild(listTechs);
+  CardTechDiv.appendChild(ulTech);
+  cardBodyDiv.appendChild(CardTechDiv);
+
+  const CardBtnDiv = document.createElement('div');
+  CardBtnDiv.className = 'card-btn';
+  const CardBtn = document.createElement('button');
+  const buttonText = document.createTextNode('See project');
+  CardBtn.appendChild(buttonText);
+  CardBtnDiv.appendChild(CardBtn);
+  cardBodyDiv.appendChild(CardBtnDiv);
+
+  cardDiv.appendChild(cardBodyDiv);
+  projectDiv.appendChild(cardDiv);
+  work.appendChild(projectDiv);
+};
+
+Projects.forEach((project) => {
+  createProject(project);
+});
+
+const cardBtn = document.querySelectorAll('.card-btn button');
+const model = document.querySelector('#model');
+
+const modelBtn = document.querySelector('#model-btn');
+const cardBtns = Array.from(cardBtn);
+
+const addDetailsToModel = (project) => {
+  const titleModel = document.querySelector('#titleModel');
+  titleModel.textContent = project.name;
+  const imageModel = document.querySelector('#imageModel');
+  imageModel.src = project.image;
+  const pragrphModel = document.querySelector('#pragrphModel');
+  pragrphModel.textContent = project.description;
+  const technologies = document.querySelector('#technologies');
+  const list = repeatList(project.technologies);
+  technologies.innerHTML = '';
+  technologies.appendChild(list);
+};
+
+cardBtns.forEach((btn, index) => {
+  btn.addEventListener('click', () => {
+    addDetailsToModel(Projects[index]);
+    model.classList.toggle('d-none');
+  });
+});
```

Comment:
[OPTIONAL] Lastly 👀  some error checking for `DOM` elements: When creating or modifying `DOM` elements, you could add checks to ensure that the required elements actually exist in the document before attempting to manipulate them. 🤔  For example, you could check that the `work` element exists before appending new projects to it, or that the `model` and `model-btn` elements exist before adding event listeners to them. 🤓